Packs/Base/Scripts/GetIndicatorsByQuery/GetIndicatorsByQuery.py

```diff
@@ -0,0 +1,143 @@
+import hashlib
+
+from CommonServerPython import *
+
+PAGE_SIZE = 500
+
+RANDOM_UUID = str(demisto.args().get('addRandomSeed', '').encode('utf8'))
+# Memo for key matching
+CACHE = {}  # type: ignore
+
+
+def hash_value(simple_value):
+    if not isinstance(simple_value, str):
+        simple_value = str(simple_value)
+    if simple_value.lower() in ["none", "null"]:
+        return None
+    return hashlib.md5(simple_value.encode('utf8') + RANDOM_UUID.encode('utf8')).hexdigest()
+
+
+def pattern_match(pattern, s):
+    regex = re.compile(pattern.replace("*", ".*"))
+    return re.match(regex, s) is not None
+
+
+def is_key_match_fields_to_hash(key, fields_to_hash):
+    if key is None:
+        return False
+
+    if key in CACHE:
+        return CACHE[key]
+    for field in fields_to_hash:
+        if pattern_match(field, key):
+            CACHE[key] = True
+            return True
+    return False
+
+
+def hash_multiple(value, fields_to_hash, to_hash=False):
+    if isinstance(value, list):
+        if not value:
+            return []
+        else:
+            return list(map(lambda x: hash_multiple(x, fields_to_hash, to_hash), value))
```

Comment:
it will have the same result...
```suggestion
        return list(map(lambda x: hash_multiple(x, fields_to_hash, to_hash), value))
```